docs/source/release_notes.rst

```diff
@@ -6,7 +6,8 @@ Release Notes
 Future Release
 ==============
     * Enhancements
-        * Add ``DayOfYear``, ``DaysInMonth``, ``Quarter`` , ``IsLeapYear`` , ``IsQuarterEnd`` , ``IsQuarterStart`` transform primitives (:pr:`2110`, :pr:`2117`)
+        * Add ``DayOfYear``, ``DaysInMonth``, ``Quarter`` , ``IsLeapYear`` , ``IsQuarterEnd`` , 
```

Comment:
We can separate this in 2-3 lines. My suggestion was to just not do 1 primitive per line.